R/render.r

```diff
@@ -42,6 +42,12 @@ render_page <- function(pkg = ".", name, data, path = "", depth = NULL, quiet =
 
   data$footer <- pkgdown_footer(data, pkg)
 
+  # Dependencies for head
+  data$headdeps <- data_deps(pkg = pkg, depth = depth)
+
+  # Potential opt-out of syntax highlighting CSS
+  data$needs_highlight_css <- !isFALSE(pkg$meta[["template"]]$params$highlightcss)
```

Comment:
we could pass the code background color via a bootstrap variable but it'd make it harder to opt-out of all the syntax highlighting styling at once.